configuration.rst

```diff
@@ -599,6 +599,15 @@ The values of these options are resolved at runtime (only once per request, to
 not impact performance) so you can change the application behavior without having
 to clear the cache.
 
+.. caution::
+
+    Environment variables are designed to store information that can
+    dynamically change in a production environment, such as an API
+    key that may have an expiration date. This information could be updated
+    without having to redeploy the application nor clear the cache. Thus, not
```

Comment:
```suggestion
    without having to redeploy the application. Thus, not
```

it is already document in the paragraph above `without having to clear the cache.`

Reply:
Removed :+1: